test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -1018,15 +1018,18 @@ public void close()
         }
     }
 
-    public static class DTestNodeTool extends NodeTool {
+    public static class DTestNodeTool extends NodeTool implements AutoCloseable
```

Comment:
this implements AutoCloseable so we can use it in try-with-resources in Instance above